install/kubernetes/helm/istio/charts/pilot/templates/deployment.yaml

```diff
@@ -58,9 +58,9 @@ spec:
 {{- end}}
           - --domain
           - {{ .Values.global.proxy.clusterDomain }}
-{{- if .Values.global.oneNamespace }}
-          - "-a"
-          - {{ .Release.Namespace }}
+{{- if .Values.appNamespace }}
```

Comment:
Please add this to the new installer ( and I suspect for demo installer it is not needed ).
When you do - please make sure you add a check on version or turn it into env variable.

The operator (and new installer) must be able to handle pervious version, so it can manage the upgrade.
Any added flag (or mesh config) breaks some cases. 

I don't mind if people really want to add new flags - but they should also add a test for upgrade/downgrade with kubeclt rolling upgrades, and a test that installer can be used with for upgrades. Or at least manually test.